src/components/DropZone/tests/DropZone.test.tsx

```diff
@@ -71,7 +72,9 @@ describe('<DropZone />', () => {
   it('calls the onDrop callback with files when a drop event is fired on document', () => {
     mountWithAppProvider(<DropZone dropOnPage onDrop={spy} />);
     const event = createEvent('drop', files);
-    document.dispatchEvent(event);
+    act(() => {
+      document.dispatchEvent(event);
+    });
```

Comment:
I'm curious why this is needed now?

Reply:
[ ...that makes sure all updates related to these “units” have been processed and applied to the DOM before you make any assertions...](https://reactjs.org/docs/testing-recipes.html#act)

You'll get an error in the console if you didn't have it.